src/app/register/supplier/FormComponent.tsx

```diff
@@ -178,7 +218,7 @@ const FormComponent = () => {
     </Formik>
     )
   }
-  const StepTwo: any = () => {
+  const StepTwo : any = () => {
```

Comment:
```suggestion
  const StepTwo : React.FC<> = () => {
```

Reply:
if i remove : any, they show me this message:

  Type '{ key: number; data: { name: string; document: string; email: string; password: string; supplierCategory: string; isSupplier: boolean; address: { street: string; number: string; neighborhood: string; city: string; state: string; zipcode: string; }; }; }' is not assignable to type 'IntrinsicAttributes'.
  Property 'data' does not exist on type 'IntrinsicAttributes'.